apps/explorer/src/pages/transaction-result/programmable-transaction-view/InputsCard.tsx

```diff
@@ -71,10 +71,22 @@ export function InputsCard({ inputs }: InputsCardProps): JSX.Element | null {
                             // Silent error
                         }
 
+                        let parsedAddress: string | null = null;
+                        try {
+                            if (parsedVector) {
+                                const hex = toHEX(new Uint8Array(parsedVector));
+                                if (hex.length == 66) {
```

Comment:
ethereum addresses have a different length, and for iota addresses we could use `isValidIotaAddress` from the sdk